Firmware/skr-mini-E3-v2.0.cfg

```diff
@@ -291,18 +282,6 @@ gcode:
    G1 E-40 F1800                  ; retract some, but not too much or it will jam
    M82                            ; set extruder to absolute
 
-##   Sensor Types
-##   "Trianglelab NTC100K B3950" (Beta 3950 used in LDO kits)
-##   "EPCOS 100K B57560G104F"
-##   "ATC Semitec 104GT-2"
-##   "Generic 3950"
-##   "Honeywell 100K 135-104LAG-J01"
-##   "NTC 100K MGB18-104F39050L32" (Keenovo Heater Pad)
-##   "AD595"
-##   "PT100 INA826"
-##   "PT1000"
-##   For more information: https://www.klipper3d.org/Config_Reference.html#temperature_sensor
-
 ## Footnote about Beta 3950:
```

Comment:
These notes should be removed if the custom thermistor definition is also being removed.